sweagent/environment/swe_env.py

```diff
@@ -749,6 +751,7 @@ def _check_syntax(self, input: str) -> tuple[str, bool]:
         output = self._communicate(f"/bin/bash -n <<'EOF'\n{input}\nEOF\n")
         return output, self.returncode == 0
 
+    @instrument("SWEEnv#communicate", params=["input", "timeout_duration"])
```

Comment:
usually this is done by adding the parent span for the callers.  I was thinking of bulking up the trace a bit anyway, because there's still quite a bit of missing information.

Reply:
will follow up PR this part.